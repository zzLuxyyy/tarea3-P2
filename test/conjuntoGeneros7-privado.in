crearConjuntoGeneros 8
esVacioConjuntoGeneros
cardinalConjuntoGeneros
imprimirConjuntoGeneros
insertarConjuntoGeneros 0
insertarConjuntoGeneros 2
insertarConjuntoGeneros 3
insertarConjuntoGeneros 5
insertarConjuntoGeneros 7
imprimirConjuntoGeneros
cardinalConjuntoGeneros
perteneceConjuntoGeneros 3
perteneceConjuntoGeneros 4
borrarDeConjuntoGeneros 5
perteneceConjuntoGeneros 5
cardinalConjuntoGeneros
imprimirConjuntoGeneros
borrarDeConjuntoGeneros 1
cardinalConjuntoGeneros
unionConjuntoGeneros 4 1 2 3 6
interseccionConjuntoGeneros 4 1 2 3 6
diferenciaConjuntoGeneros 4 1 2 3 6
borrarDeConjuntoGeneros 0
borrarDeConjuntoGeneros 2
borrarDeConjuntoGeneros 3
borrarDeConjuntoGeneros 7
esVacioConjuntoGeneros
cardinalConjuntoGeneros
imprimirConjuntoGeneros
unionConjuntoGeneros 2 1 5
interseccionConjuntoGeneros 2 1 5
diferenciaConjuntoGeneros 2 1 5
liberarConjuntoGeneros
Fin