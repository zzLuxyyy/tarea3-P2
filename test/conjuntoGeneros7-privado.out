1>Se creó conjunto generos con cantMax 8 exitosamente.
2>El conjunto de generos es vacío.
3>El cardinal del conjunto es: 0.
4>Conjunto de generos:

5>Se ejecutó 'insertarTConjuntoGeneros con genero de id 0 exitosamente.
6>Se ejecutó 'insertarTConjuntoGeneros con genero de id 2 exitosamente.
7>Se ejecutó 'insertarTConjuntoGeneros con genero de id 3 exitosamente.
8>Se ejecutó 'insertarTConjuntoGeneros con genero de id 5 exitosamente.
9>Se ejecutó 'insertarTConjuntoGeneros con genero de id 7 exitosamente.
10>Conjunto de generos:
0 2 3 5 7
11>El cardinal del conjunto es: 5.
12>El genero de id 3 pertenece al conjunto.
13>El genero de id 4 NO pertenece al conjunto.
14>Se ejecutó 'borrarDeTConjuntoGeneros con genero de id 5 exitosamente.
15>El genero de id 5 NO pertenece al conjunto.
16>El cardinal del conjunto es: 4.
17>Conjunto de generos:
0 2 3 7
18>Se ejecutó 'borrarDeTConjuntoGeneros con genero de id 1 exitosamente.
19>El cardinal del conjunto es: 4.
20>Conjunto de generos unión:
0 1 2 3 6 7
21>Conjunto de generos intersección:
2 3
22>Conjunto de generos diferencia:
0 7
23>Se ejecutó 'borrarDeTConjuntoGeneros con genero de id 0 exitosamente.
24>Se ejecutó 'borrarDeTConjuntoGeneros con genero de id 2 exitosamente.
25>Se ejecutó 'borrarDeTConjuntoGeneros con genero de id 3 exitosamente.
26>Se ejecutó 'borrarDeTConjuntoGeneros con genero de id 7 exitosamente.
27>El conjunto de generos es vacío.
28>El cardinal del conjunto es: 0.
29>Conjunto de generos:

30>Conjunto de generos unión:
1 5
31>Conjunto de generos intersección:

32>Conjunto de generos diferencia:

33>Se ejecutó 'liberarTConjuntoGeneros' exitosamente.
34>Fin.
